### PR TITLE
Fix intermittent failures on CI

### PR DESCRIPTION
**Pull request by paulbalandan**
Trying to fix intermittent failures on CI by using the workflows on coding-standard

### CODE REVIEW

File: .github/workflows/auto-review.yml [hunk 1]
```diff
@@ -56,8 +54,14 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ env.COMPOSER_CACHE_FILES_DIR }}
-          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
-          restore-keys: ${{ runner.os }}-composer-
+          key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}
+          restore-keys: |
+            composer-${{ runner.os }}-${{ matrix.php-version }}-
+            composer-${{ runner.os }}-
+            composer-
```
Comment (kenjis) [inline .github/workflows/auto-review.yml:61]: Why do we need this change?

Comment (paulbalandan) [reply .github/workflows/auto-review.yml]: That was copy-paste. Reverted.

**Review by kenjis — APPROVED** [review]

**Comment by paulbalandan** [comment]
Thank you, @kenjis, for the review!

**Comment by paulbalandan** [comment]
I will wait until tomorrow (before I make a patch release) when the scheduled run finishes to see if this works as intended.

**Comment by paulbalandan** [comment]
It seems pushes to the `develop` causes the authentication errors. But PRs do not error. 🤔

**Pull request merged**